### PR TITLE
Add MWeb

### PR DESCRIPTION
**Pull request by oimotis**
MWeb - Pro Markdown writing, note taking and static blog generator App

Official Site: http://www.mweb.im/
Mac App Store: https://itunes.apple.com/us/app/mweb-pro-markdown-writing/id954188948?ls=1&mt=12

### CODE REVIEW

**Comment by iCHAIT** [comment]
@dkhamsing @herrbischoff  Thoughts on this?

**Comment by iCHAIT** [comment]
@dkhamsing @herrbischoff thoughts on this?

File: README.md [hunk 1]
```diff
@@ -191,6 +191,7 @@
 - [Karabiner](https://pqrs.org/osx/karabiner/) - A powerful keyboard customizer. [![Open-Source Software][OSS Icon]](https://github.com/tekezo/Karabiner) ![Freeware][Freeware Icon]
 - [Keytty](http://keytty.com) - Enables you to control your mouse with a few key strokes. Mouse Keys Alternative.
 - [LaunchBar](https://www.obdev.at/products/launchbar/index.html) - Start applications, navigate folders, manipulate files, control your Mac and much more just by using the keyboard.
+- [MWeb](http://www.mweb.im/) - Pro Markdown writing, note taking and static blog generator App
```
Comment (oimotis) [inline README.md]: Sorry!

**Comment by dkhamsing** [comment]
looks good 
we have many markdown apps but the blog generation is novel.. seems like a pretty specific need 

cost is $8 💰

**Comment by herrbischoff** [comment]
Well... I don't like the idea of a "Markdown IDE" of sorts. Maybe that's just me though. It's not compelling enough for me to try out. A really professional writing tool with Markdown support would be Ulysses (http://ulyssesapp.com), from which this application seems to take major cues.

**Comment by oimotis** [comment]
@herrbischoff 

I think MWeb is not a "Markdown IDE",  it's more friendly for "Pro" Markdown users, please check the gif:
demo

MWeb also can publish the document to WordPress, Blogger, Medium and Evernote.

**Comment by herrbischoff** [comment]
For now I add the label "needs endorsement" and we'll wait for others to comment on this.

**Comment by havthgem** [comment]
This is good, although it doesn't look beauty (Maybe just that I dislike it). But, MWeb is still powerful.

**Comment by dkhamsing** [comment]
@herrbischoff shall we close this?

**Comment by herrbischoff** [comment]
@dkhamsing: Yes.

**Pull request closed without merging**